mmv1/templates/terraform/examples/firebase_app_hosting_backend_full.tf.tmpl

```diff
@@ -0,0 +1,66 @@
+resource "google_firebase_app_hosting_backend" "example" {
+  project          = "{{index $.TestEnvVars "project_id"}}"
+
+  # Choose the region closest to your users
+  location         = "us-central1"
+  backend_id       = "{{index $.Vars "backend_id"}}"
+  app_id           = "1:0000000000:web:674cde32020e16fbce9dbd"
```

Comment:
You did confirm on last review that these examples looked good for user consumption, but just to double check, this `app_id` is expected to be used by regular users? Sorry to ask again, it just looks very specific to an environment.